From c24044955c915c5d24a57137e71406af33f92d6e Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 22 May 2024 21:47:06 +0200
Subject: [PATCH 1129/1141] arm64: dts: st: add disabled spi6 node into
 stm32mp257f-dk

Add a disabled spi6 node for the spi6 instance available via
the GPIO expansion connector on the stm32mp257f-dk board.

Change-Id: I85d61bc7c0d33f647ccefd7ac5a3f319a4fdd953
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/381854
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 7 +++++++
 1 file changed, 7 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -676,6 +676,13 @@
 	};
 };
 
+&spi6 {
+	pinctrl-names = "default", "sleep";
+	pinctrl-0 = <&spi6_pins_a>;
+	pinctrl-1 = <&spi6_sleep_pins_a>;
+	status = "disabled";
+};
+
 /* Bluetooth */
 &usart1 {
 	pinctrl-names = "default", "sleep", "idle";
